go-python.h

```diff
@@ -1,7 +1,11 @@
 #ifndef GOPYTHON_GOPYTHON_H
 #define GOPYTHON_GOPYTHON_H 1
 
+#ifdef __APPLE__
+#include "Python/Python.h"
+#else
 #include "Python.h"
+#endif
 
 #include "frameobject.h"
```

Comment:
shouldn't all the other "internal" python-related headers be picked up from under `"Python/..."` as well?